src/index.js

```diff
@@ -585,6 +585,7 @@ export default class Trigger extends React.Component {
 
   savePopup = (node) => {
     this._component = node;
+    this.props.afterPopupVisibleChange(this.props.visible)
```

Comment:
`afterPopupVisibleChange` should only trigger when visible change. Init of popup ref should not trigger this.